demisto_sdk/__main__.py

```diff
@@ -368,8 +368,7 @@ def secrets(config, **kwargs):
         is_circle=kwargs['post_commit'],
         ignore_entropy=kwargs['ignore_entropy'],
         white_list_path=kwargs['whitelist'],
-        input_path=kwargs.get('input'),
-        prev_ver=kwargs.get('prev_ver')
```

Comment:
why did you take out this parameter? this is used in the 4.5 feature branch.

Reply:
Didn't remove it. Must not be updated with master. Will pull